exercises/concept/freelancer-rates/freelancer-rates.js

```diff
@@ -1,8 +1,23 @@
 // @ts-check
 //
-// The line above enables type checking for this file. Various IDEs interpret
-// the @ts-check directive. It will give you helpful autocompletion when
-// implementing this exercise.
+// ☝🏽 The line above enables type checking for this file. Various IDEs interpret
+// the @ts-check directive. It will give you helpful autocompletion on the web
+// and supported IDEs when implementing this exercise. You don't need to
+// understand types, JSDoc, or TypeScript in order to complete this JavaScript
+// exercise, and can completely ignore this comment block and directive.
+
+// 👋🏽 Hi again!
+//
+// A quick reminder about exercise stubs:
+//
+// 💡 You're allowed to completely clear any stub before you get started. Often
+// we recommend using the stub, because they are already set-up correctly to
+// work with the tests, which you can find in ./freelancer-rates.spec.js.
+//
+// 💡 You don't need to write JSDoc comment blocks yourself; it is not expected
+// in idiomatic JavaScript, but some companies and style-guides do enforce them.
+//
+// Get those rates calculated!
```

Comment:
This comment seems to be in `exemplar.js` and duplicated here.  Should it that exemplar does not have these comments?

Reply:
The exemplar solution would have used the same stub you start from. Haven't decided if it's better to leave the comments or to remove them.